src/main/java/pikabot/TaskList.java

```diff
@@ -0,0 +1,104 @@
+package pikabot;
+
+import java.util.ArrayList;
+
+import pikabot.task.Task;
+
+/**
+ * Contains tasks and performs operations to add, delete, mark, and find tasks.
+ */
+public class TaskList {
+
+    protected ArrayList<Task> taskList;
```

Comment:
Might be confusing for "TaskList" class to have an attribute "taskList"? "taskList" can be changed to "tasks"